presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -102,8 +102,67 @@ Property Name                                      Description
 ``hive.immutable-partitions``                      Can new data be inserted into existing partitions?           ``false``
 
 ``hive.max-partitions-per-writers``                Maximum number of partitions per writer.                     100
+================================================== ============================================================ ==========
+
+Amazon S3 Configuration
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The Hive connector also allows querying data in Hive where data files are stored in Amazon S3. 
+The SQL support for S3 tables remains the same as for HDFS tables.
+
+For accessing tables stored in S3 the mandatory configuration properties required
```

Comment:
It would be simpler to say:
To access tables stored in S3, you must specify the AWS credential properties....